.github/workflows/build-push-docker.yml

```diff
@@ -0,0 +1,65 @@
+name: Build and push Docker image
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "*"
+  pull_request:
+    branches:
+      - main
```

Comment:
Hmm, why do want to run this for PRs?

Reply:
To see that workflow works before merging it :) Will remove soon

Reply:
Ah lol makes sense